static-site/posts/updates/2019.3.11-update.udon

```diff
@@ -0,0 +1,31 @@
+:-  :~  date/'~2019.3.20'
+        title/'~2019.3.11 Update'
```

Comment:
I don't think this date is right. Shouldn't it be `~2019.3.20` In all instances?

Reply:
We date the weekly update to the week that it's documenting (week of ~2019.3.11). I could be convinced that this is confusing, but it's the pattern.